lib/txdb/handlers/triggers/handler.rb

```diff
@@ -24,26 +24,17 @@ def handle_safely
           respond_with_error(500, "Internal server error: #{e.message}", e)
         end
 
-        def databases
-          @databases ||= if database_name = request.params['database']
-            Txdb::Config.databases.select do |database|
-              database.database == database_name
-            end
-          else
-            Txdb::Config.databases
+        def database
+          @database ||= Txdb::Config.databases.find do |database|
+            database.database == request.params['database']
```

Comment:
Possible to rename `database.database` to `database.name`? Block param also has same name as `__method__`, maybe use `config` instead?